### PR TITLE
completed header, footer, and testing coverage for both

### PR DESCRIPTION
**Pull request by cherrythefool**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,10 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
+body {
```
Comment (TsunamiCode5) [inline src/App.css]: Is this an empty CSS object?

**Review by TsunamiCode5 — COMMENTED** [review]
Also, I am seeing a placeholder.md that is empty. Are we planning to use it later or is this not supposed to be pushed up?

File: src/App.css (hunk 1 above)
Comment (cherrythefool) [reply src/App.css]: yes, we opened it since we know we'll use it once we have more content on the page. we'll add something in it or take it out!

Comment (TsunamiCode5) [reply src/App.css]: ✅

**Review by TsunamiCode5 — APPROVED** [review]

**Pull request merged**